app/Metrics/App/WikiMetrics.php

```diff
@@ -210,4 +212,17 @@ private function getNumberOfEntities(): array {
 
         return $result;
     }
+
+    protected function getTotalUserCount() {
+        $wikiDb = $this->wiki->wikiDb;
+        $tableUser = $wikiDb->name . '.' . $wikiDb->prefix . '_user';
+        $query = "SELECT COUNT(*) AS total_users FROM $tableUser";
+        $manager = app()->db;
+        $manager->purge('mw');
+        $conn = $manager->connection('mw');
+        $pdo = $conn->getPdo();
+        $result = $pdo->query($query)->fetchAll(PDO::FETCH_ASSOC);
```

Comment:
no need for fetchAll() since we only need 1 row
```suggestion
        $result = $pdo->query($query)->fetch(PDO::FETCH_ASSOC);
```

Reply:
Yes, thank you for spotting that